projects/laji/src/app/+project-form/results/biomon-result/biomon-result-statistics/biomon-result-statistics.component.ts

```diff
@@ -0,0 +1,94 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { CollectionApi } from 'projects/laji/src/app/shared/api/CollectionApi';
+import { WarehouseApi } from 'projects/laji/src/app/shared/api/WarehouseApi';
+import { toHtmlSelectElement } from 'projects/laji/src/app/shared/service/html-element.service';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { map, switchMap, tap } from 'rxjs/operators';
+
+interface Row {
+  observer: string;
+  gatherings: number;
+}
+
+interface QueryResult {
+  results: {
+    aggregateBy: {
+      'gathering.team.memberName': string;
+    };
+    count: number;
+  }[];
+}
+
+@Component({
+  selector: 'laji-biomon-result-statistics',
+  templateUrl: './biomon-result-statistics.component.html',
+  styleUrls: ['./biomon-result-statistics.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class BiomonResultStatisticsComponent implements OnInit {
+  readonly collection$ = new BehaviorSubject<string>('');
+  @Input() set collection(v: string) { this.collection$.next(v); };
```

Comment:
This pattern is interesting and I don't necessarily see any problems with it, however, it would be more angular-like to implement the `OnChanges` interface instead of using a setter. Not sure if you need to change this, just a thought.

Reply:
I leave it for now, but I might check it out at a later time. I could see if I can implement this to the next result service and change it here too.